documentation/src/main/asciidoc/topics/con_cache_encoding.adoc

```diff
@@ -0,0 +1,41 @@
+[id='encoding']
+= Cache encoding
+Encoding is the format, or media type, that {brandname} uses to store entries (key/value pairs) in caches.
+
+[discrete]
+== Remote caches
+
+{brandname} Server stores entries in remote caches without any media type information by default.
+
+Encoding configures how {brandname} stores entries in a remote cache.
+If you do not configure encoding for remote caches then {brandname} Server stores entries however clients provide them.
```

Comment:
I'm not sure of a better way to word this, but this isn't really correct. If no encoding is specified, then the key/value will be stored as a generic `byte[]` on the server. This isn't necessarily how the clients provided them, as in the case of the HR client, it's likely the key/values were POJOs that were marshalled into `byte[]` so that they could be sent to the server. So the phrase "however clients provide them" may cause the user to think that the key/value is stored as a POJO on the server-side.

Reply:
How about 

"If you do not configure encoding for remote caches then {brandname} Server stores keys and values as generic `byte[]`.
When clients request entries from caches without any encoding, {brandname} Server returns keys and values as they are stored in the remote cache."

I'm trying to clarify that - without encoding - clients might not be able to handle the entries that the server returns. Would that make sense?

Reply:
Your suggestion is a big improvement I think.

Reply:
great. thanks!